x-pack/plugins/enterprise_search/public/applications/enterprise_search/index.scss

```diff
@@ -1,41 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-.enterpriseSearchOverview {
-  padding-top: 78px;
-  background-image: url('./assets/bg_enterprise_search.png');
```

Comment:
Now that we're no longer using this image, do you mind deleting the actual asset as well? https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/enterprise_search/assets/bg_enterprise_search.png